index.d.ts

```diff
@@ -120,11 +120,11 @@ export interface AssertContext {
 	/**
 	 * Assert that value is deep equal to expected.
 	 */
-	same<U>(value: U, expected: U, message?: string): void;
+	deepEqual<U>(value: U, expected: U, message?: string): void;
```

Comment:
I think that's the best way to handle this situations as well. It will indeed throw a compile error because it could not find the `same` method.